src/reference/asciidoc/mail.adoc

```diff
@@ -143,6 +143,10 @@ In this case, the only header populated is the mentioned above `IntegrationMessa
 Starting with version 5.5.11, the folder is closed automatically after `AbstractMailReceiver.receive()` if no messages received or all of them are filtered out independently of the `autoCloseFolder` flag.
 In this case there is nothing to produce downstream for possible logic around `IntegrationMessageHeaderAccessor.CLOSEABLE_RESOURCE` header.
 
+Starting with version 6.0.5, the `ImapIdleChannelAdapter` doesn't do an asynchronous message publishing anymore.
+This is necessary to block the idle listener loop for messages processing downstream (e.g. with big attachments) for which the mail folder must remain opened.
+If an async hand-off is appropriate, an `ExecutorChannel` as an output channel of this channel adapter is a recommended way from now on.
```

Comment:
```suggestion
Starting with version 6.0.5, the `ImapIdleChannelAdapter` no longer performs asynchronous message publishing.
This is necessary to block the idle listener loop for message processing downstream (e.g. with big attachments) because the mail folder must remain open.
If an async hand-off is required, an `ExecutorChannel` can be used as the output channel of this channel adapter.
```